low-level/framework/base/sspl_constants.py

```diff
@@ -252,6 +252,40 @@
 
 SSPL_CONFIGS = ['log_level', 'cli_type', 'sspl_log_file_path', 'cluster_id', 'storage_enclosure', 'setup', 'operating_system']
 
+# SSPL python 3rd party package dependencies
+pip3s_packages_main = {
```

Comment:
Can we rename to pip3_3ps_pkgs_main

Reply:
Sure. May I know what does `3ps` mean to?

Reply:
Done.